src/components/Notification/Notification.tsx

```diff
@@ -376,6 +376,8 @@ const NotificationDropdown: React.FC = () => {
     useState(false)
   const [notification, setNotification] = useState<Notification>()
 
+  const dropdownRef = useRef<HTMLDivElement>(null)
+
   const toggleDropdown = () => {
     setIsOpen(!isOpen)
```

Comment:
```suggestion
  const toggleDropdown = (e: React.MouseEvent) => {
    e.stopPropagation()
    setIsOpen(!isOpen)
```
[P1] 이벤트의 자식 전달을 제거합니다.